packages/devtools_app/lib/src/profiler/flutter/cpu_profiler.dart

```diff
@@ -80,23 +129,19 @@ class CpuProfiler extends StatelessWidget {
   List<Widget> _buildProfilerViews() {
     final cpuFlameChart = LayoutBuilder(builder: (context, constraints) {
       return CpuProfileFlameChart(
-        data,
+        widget.data,
         // TODO(kenz): remove * 2 once zooming is possible. This is so that we can
         // test horizontal scrolling functionality.
         width: constraints.maxWidth * 2,
-        selected: selectedStackFrame,
-        onSelected: (sf) => onStackFrameSelected(sf),
+        selected: widget.selectedStackFrame,
+        onSelected: (sf) => widget.onStackFrameSelected(sf),
       );
     });
 
     // TODO(kenz): tree table is extremely slow with large data set. It should
     // be optimized before including in the profiler.
```

Comment:
if this is fixed, please remove the todo

Reply:
Done.